### ISSUE TITLE
[eas build] Build failed

### ISSUE THREAD

**Issue by pCyril** [expo/expo#22506]
### Summary

Hello,
I have an issue when fastlane compiling expo-image convertion module

› Compiling expo-image Pods/SDWebImageAVIFCoder » Conversion.m

❌  (ios/Pods/SDWebImageAVIFCoder/SDWebImageAVIFCoder/Classes/Conversion.m:158:28)

  156 |     state->biasY = (state->yuvRange == AVIF_RANGE_LIMITED) ? (float)(16 << (state->yuvDepth - 8)) : 0.0f;
  157 |     state->biasUV = (float)(1 << (state->yuvDepth - 1));
> 158 |     state->biasA = (image->alphaRange == AVIF_RANGE_LIMITED) ? (float)(16 << (state->yuvDepth - 8)) : 0.0f;
      |                            ^ no member named 'alphaRange' in 'struct avifImage'; did you mean 'alphaPlane'?
  159 |     state->rangeY = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
  160 |     state->rangeUV = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (224 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
  161 |     state->rangeA = (float)((image->alphaRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);

❌  (ios/Pods/SDWebImageAVIFCoder/SDWebImageAVIFCoder/Classes/Conversion.m:158:12)

  156 |     state->biasY = (state->yuvRange == AVIF_RANGE_LIMITED) ? (float)(16 << (state->yuvDepth - 8)) : 0.0f;
  157 |     state->biasUV = (float)(1 << (state->yuvDepth - 1));
> 158 |     state->biasA = (image->alphaRange == AVIF_RANGE_LIMITED) ? (float)(16 << (state->yuvDepth - 8)) : 0.0f;
      |            ^ no member named 'biasA' in 'struct avifReformatState'; did you mean 'biasY'?
  159 |     state->rangeY = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
  160 |     state->rangeUV = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (224 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
  161 |     state->rangeA = (float)((image->alphaRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);

❌  (ios/Pods/SDWebImageAVIFCoder/SDWebImageAVIFCoder/Classes/Conversion.m:161:37)

  159 |     state->rangeY = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
  160 |     state->rangeUV = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (224 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
> 161 |     state->rangeA = (float)((image->alphaRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
      |                                     ^ no member named 'alphaRange' in 'struct avifImage'; did you mean 'alphaPlane'?
  162 | 
  163 |     uint32_t cpCount = 1 << image->depth;
  164 |     if (state->mode == AVIF_REFORMAT_MODE_IDENTITY) {

❌  (ios/Pods/SDWebImageAVIFCoder/SDWebImageAVIFCoder/Classes/Conversion.m:161:12)

  159 |     state->rangeY = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
  160 |     state->rangeUV = (float)((state->yuvRange == AVIF_RANGE_LIMITED) ? (224 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
> 161 |     state->rangeA = (float)((image->alphaRange == AVIF_RANGE_LIMITED) ? (219 << (state->yuvDepth - 8)) : state->yuvMaxChannel);
      |            ^ no member named 'rangeA' in 'struct avifReformatState'; did you mean 'rangeY'?
  162 | 
  163 |     uint32_t cpCount = 1 << image->depth;
  164 |     if (state->mode == AVIF_REFORMAT_MODE_IDENTITY) {

❌  (ios/Pods/SDWebImageAVIFCoder/SDWebImageAVIFCoder/Classes/Conversion.m:570:18)

  568 |     if(hasAlpha) { // alpha
  569 |         vImage_Buffer alphaBuffer = {0};
> 570 |         if(avif->alphaRange == AVIF_RANGE_LIMITED) {
      |                  ^ no member named 'alphaRange' in 'struct avifImage'; did you mean 'alphaPlane'?
  571 |             float* floatAlphaBufferData = NULL;
  572 |             floatAlphaBufferData = calloc(avif->width * avif->height, sizeof(float));
  573 |             scaledAlphaBufferData = calloc(avif->width * avif->height, sizeof(uint8_t));

❌  (ios/Pods/SDWebImageAVIFCoder/SDWebImageAVIFCoder/Classes/Conversion.m:865:22)

  863 |         float rangeMax = 0.0f;
  864 |         if(avif->depth == 10) {
> 865 |             if(avif->alphaRange == AVIF_RANGE_LIMITED) {
      |                      ^ no member named 'alphaRange' in 'struct avifImage'; did you mean 'alphaPlane'?
  866 |                 offset = 64.0f;
  867 |                 rangeMax = 940.0f;
  868 |             } else {

❌  (ios/Pods/SDWebImageAVIFCoder/SDWebImageAVIFCoder/Classes/Conversion.m:873:22)

  871 |             }
  872 |         } else if(avif->depth == 12) {
> 873 |             if(avif->alphaRange == AVIF_RANGE_LIMITED) {
      |                      ^ no member named 'alphaRange' in 'struct avifImage'; did you mean 'alphaPlane'?
  874 |                 offset = 256.0f;
  875 |                 rangeMax = 3760.0f;
  876 |             } else {

### Managed or bare workflow?

managed

### What platform(s) does this occur on?

iOS

### Environment

expo-env-info 1.0.5 environment info:
    System:
      OS: Linux 5.19 Ubuntu 22.04.2 LTS 22.04.2 LTS (Jammy Jellyfish)
      Shell: 5.1.16 - /bin/bash
    Binaries:
      Node: 18.16.0 - /bin/node
      Yarn: 1.22.4 - /usr/local/bin/yarn
      npm: 9.5.1 - /bin/npm
    npmPackages:
      expo: ^48.0.0 => 48.0.4 
      react: 18.2.0 => 18.2.0 
      react-dom: 18.2.0 => 18.2.0 
      react-native: 0.71.8 => 0.71.8 
      react-native-web: ~0.18.7 => 0.18.10 
    npmGlobalPackages:
      eas-cli: 3.12.1
      expo-cli: 6.3.7
    Expo Workflow: managed

### Reproducible demo

I don't have any issue with my code it works latest build was successfull 12 days ago
I can give you my package.json

{
  "name": "********",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "web": "expo start --web",
    "eject": "expo eject"
  },
  "dependencies": {
    "@expo-google-fonts/fira-sans": "^0.2.3",
    "@expo/vector-icons": "^13.0.0",
    "@gorhom/bottom-sheet": "^4",
    "@ide/backoff": "^1.0.0",
    "@miblanchard/react-native-slider": "^2.1.0",
    "@react-native-async-storage/async-storage": "1.17.11",
    "@react-native-community/datetimepicker": "6.7.3",
    "@react-native-community/netinfo": "9.3.7",
    "@react-native-community/toolbar-android": "^0.2.1",
    "@react-navigation/bottom-tabs": "^6.0.9",
    "@react-navigation/native": "^6.0.6",
    "@react-navigation/native-stack": "^6.2.5",
    "@react-navigation/stack": "^6.0.11",
    "axios": "^0.24.0",
    "axios-auth-refresh": "^3.3.1",
    "base-64": "^1.0.0",
    "expo": "^48.0.0",
    "expo-application": "~5.1.1",
    "expo-auth-session": "~4.0.3",
    "expo-file-system": "~15.2.2",
    "expo-font": "~11.1.1",
    "expo-image": "~1.0.0",
    "expo-image-crop": "^1.0.4",
    "expo-image-manipulator": "~11.1.1",
    "expo-image-picker": "~14.1.1",
    "expo-intent-launcher": "~10.5.2",
    "expo-linear-gradient": "~12.1.2",
    "expo-location": "~15.1.1",
    "expo-notifications": "~0.18.1",
    "expo-random": "~13.1.1",
    "expo-secure-store": "~12.1.1",
    "expo-status-bar": "~1.4.4",
    "expo-updates": "~0.16.4",
    "expo-web-browser": "~12.1.1",
    "form-data": "^4.0.0",
    "jwt-decode": "^3.1.2",
    "lodash": "^4.17.21",
    "lottie-react-native": "5.1.4",
    "moment": "^2.29.4",
    "query-string": "^8.1.0",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-native": "0.71.8",
    "react-native-animatable": "^1.3.3",
    "react-native-blurhash": "^1.1.10",
    "react-native-geocoding": "^0.5.0",
    "react-native-gesture-handler": "~2.9.0",
    "react-native-keyboard-aware-scroll-view": "^0.9.5",
    "react-native-linear-gradient": "^2.6.2",
    "react-native-maps": "1.3.2",
    "react-native-modal-datetime-picker": "^13.1.2",
    "react-native-reanimated": "~2.14.4",
    "react-native-safe-area-context": "4.5.0",
    "react-native-screens": "~3.20.0",
    "react-native-svg": "13.4.0",
    "react-native-toast-message": "^2.1.5",
    "react-native-web": "~0.18.7",
    "react-native-webview": "11.26.0",
    "react-native-youtube-iframe": "^2.2.2",
    "react-query": "^3.38.1"
  },
  "devDependencies": {
    "@babel/core": "^7.19.3",
    "@types/react": "~18.0.27",
    "@types/react-native": "~0.70.6",
    "babel-plugin-module-resolver": "^5.0.0",
    "typescript": "^4.9.4"
  },
  "private": true
}

### Stacktrace (if a crash is involved)

You can find my build trace here
https://expo.dev/accounts/pcyril/projects/bamboche/builds/f3dc50e8-696e-43c9-aec9-d512c23c6359

**Comment by gaishimo** [comment]
@pCyril I also experienced the same problem with expo-image v1.0. I suggest you update expo-image to the latest version (1.2.3).

**Comment by pCyril** [comment]
Yes thanks i found it 
Have a nice day

**Comment by SavageWilliam** [comment]
I had this issue too! Wil try 1.2.3 today

**Comment by dereklucas** [comment]
Updating to 1.2.3 worked... until today. Now `npx expo install --check` complains that I'm not using expo-image `1.0.1`.
I assume that 1.0.1 is a backport of the dependency change, but it means that currently I can't use 1.2.3.

**Comment by pisacode** [comment]
I am currently getting errors on builds with the possible same problem 

```
[INSTALL_PODS] [!] CocoaPods could not find compatible versions for pod "SDWebImageAVIFCoder":
[INSTALL_PODS]   In Podfile:
[INSTALL_PODS]     ExpoImage (from `../node_modules/expo-image/ios`) was resolved to 1.0.1, which depends on
[INSTALL_PODS]       SDWebImageAVIFCoder (~> 0.10.0)
[INSTALL_PODS] None of your spec sources contain a spec satisfying the dependency: `SDWebImageAVIFCoder (~> 0.10.0)`.
[INSTALL_PODS] You have either:
[INSTALL_PODS]  * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
[INSTALL_PODS]  * mistyped the name or version.
[INSTALL_PODS]  * not added the source repo that hosts the Podspec to your Podfile.
[INSTALL_PODS] 
```


anyone having the same issue?

**Comment by newfylox** [comment]
Having the same issue of @pisacode and being on version 1.2.3. 
Any idea?
